hashbang/welcome

```diff
@@ -0,0 +1,11 @@
+if [ -d "${HB_CONF}" ]; then
```

Comment:
In the script that sources `hashbang/welcome`, but it's true it is now useless cruft.